src/node/desktop/src/main/args-manager.ts

```diff
@@ -131,6 +132,11 @@ export class ArgsManager {
       application.sessionStartDelaySeconds = 5;
     }
 
+    // switch for forcing the splash to show by adding a startup delay
+    if (app.commandLine.hasSwitch(kSplashForce)) {
```

Comment:
Do we want to allow users to customize the startup delay here? (also the constant name doesn't match the flag name)

Reply:
Sounds like a good thing to add!